### PR TITLE
Solved crashing problem in nomination button

### LINKED ISSUE #3520: Application crashes when nomination for deletion is pressed
**Steps to reproduce:** 
1. Install the Application.
2. Skip the Login or Signup section
3. Go to the Explore section and select one image
4. Tap the button Nominate for deletion

**System logs:**
`2020-03-15 20:11:48.076 12723-12723/fr.free.nrw.commons.beta E/AndroidRuntime: FATAL EXCEPTION: main
    Process: fr.free.nrw.commons.beta, PID: 12723
    java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
        at fr.free.nrw.commons.media.MediaDetailFragment.onDeleteButtonClicked(MediaDetailFragment.java:351)
        at fr.free.nrw.commons.media.MediaDetailFragment_ViewBinding$5.doClick(MediaDetailFragment_ViewBinding.java:90)
        at butterknife.internal.DebouncingOnClickListener.onClick(DebouncingOnClickListener.java:18)
        at android.view.View.performClick(View.java:7125)
        at android.view.View.performClickInternal(View.java:7102)
        at android.view.View.access$3500(View.java:801)
        at android.view.View$PerformClick.run(View.java:27336)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
2020-03-15 20:11:48.077 12723-12723/fr.free.nrw.commons.beta E/ACRA: ACRA caught a NullPointerException for fr.free.nrw.commons.beta
    java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
        at fr.free.nrw.commons.media.MediaDetailFragment.onDeleteButtonClicked(MediaDetailFragment.java:351)
        at fr.free.nrw.commons.media.MediaDetailFragment_ViewBinding$5.doClick(MediaDetailFragment_ViewBinding.java:90)
        at butterknife.internal.DebouncingOnClickListener.onClick(DebouncingOnClickListener.java:18)
        at android.view.View.performClick(View.java:7125)
        at android.view.View.performClickInternal(View.java:7102)
        at android.view.View.access$3500(View.java:801)
        at android.view.View$PerformClick.run(View.java:27336)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
`

**Device and Android version:** 
Device name - Pixel 3a XL
Android version - Android 10
API level - 29

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by gouri-panda**
**Description (required)**

Fixes #3520  Application crashes when the nomination for deletion is pressed

What changes did you make and why?
Added check method if the user name exists or not

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java [hunk 1]
```diff
@@ -348,67 +348,74 @@ public void onCopyWikicodeClicked(){
...
-        AlertDialog d = alert.create();
-        input.addTextChangedListener(new TextWatcher() {
-            private void handleText() {
-                final Button okButton = d.getButton(AlertDialog.BUTTON_POSITIVE);
-                if (input.getText().length() == 0) {
-                    okButton.setEnabled(false);
-                } else {
-                    okButton.setEnabled(true);
+        if (AccountUtil.getUserName(getContext())!= null) {
+            if (AccountUtil.getUserName(getContext()).equals(media.getCreator())) {
```
Comment (macgills) [inline app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java]: I would just merge these 2 if statements (if it is valid that a non logged in user can nominate things for deletion), also the null check is not in style, no space on the left hand side

Comment (gouri-panda) [reply app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java]: @macgills  Done. :)

File: .idea/codeStyles/Project.xml [hunk 2]
```diff
@@ -47,31 +47,9 @@
       <option name="SPACE_BEFORE_SUPERCLASS_COLON" value="false" />
     </Objective-C>
     <Objective-C-extensions>
-      <option name="GENERATE_INSTANCE_VARIABLES_FOR_PROPERTIES" value="ASK" />
```
Comment (macgills) [inline .idea/codeStyles/Project.xml]: revert the changes to this file please

Comment (gouri-panda) [reply .idea/codeStyles/Project.xml]: @macgills  Done. :)

**Pull request merged**